io.gitlab.Turtlico.json

```diff
@@ -97,59 +97,60 @@
+                "sha256": "06828c04a573c073a4e51c4292a27c1be4ae26621c3edc7cf9318418ce3b6d27"
+            }]
         },
         {
             "name": "python3-Pillow",
             "buildsystem": "simple",
             "build-commands": [
                 "LDFLAGS=-L`echo /usr/lib/${FLATPAK_ARCH}-linux-*` pip3 install --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} Pillow"
             ],
-            "sources": [
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/2b/06/93bf1626ef36815010e971a5ce90f49919d84ab5d2fa310329f843a74bc1/Pillow-8.0.1.tar.gz",
-                    "sha256": "11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e"
-                }
-            ]
+            "sources": [{
+                "type": "file",
+                "url": "https://files.pythonhosted.org/packages/7d/2a/2fc11b54e2742db06297f7fa7f420a0e3069fdcf0e4b57dfec33f0b08622/Pillow-8.4.0.tar.gz",
+                "sha256": "b8e2f83c56e141920c39464b852de3719dfbfb6e3c99a2d8da0edf4fb33176ed"
+            }]
         },
         {
             "name": "python3-gpiozero",
             "buildsystem": "simple",
             "build-commands": [
                 "pip3 install --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} gpiozero"
             ],
-            "sources": [
-                {
+            "sources": [{
                     "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/cc/a2/718e78485d0a42c3c2971e0d4a0eb9878bd1121cd640f0773de0695c858e/colorzero-1.1-py2.py3-none-any.whl",
-                    "sha256": "e3c36d15b293de2b2f77ff54a5bd243fffac941ed0a5332d0697a6612a26a0a3"
+                    "url": "https://files.pythonhosted.org/packages/b3/ca/688824a06e8c4d04c7d2fd2af2d8da27bed51af20ee5f094154e1d680334/colorzero-2.0.tar.gz",
+                    "sha256": "e7d5a5c26cd0dc37b164ebefc609f388de24f8593b659191e12d85f8f9d5eb58"
                 },
                 {
                     "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/5a/06/e3440c452a0c2ba1f12527c12c5676b158f2591ac6c43979016f21f71b2a/gpiozero-1.5.1-py2.py3-none-any.whl",
-                    "sha256": "101044038e71cec8be2fa841d4cea9e50e7962e9fb0287f465298b59e3f06594"
+                    "url": "https://files.pythonhosted.org/packages/86/e1/88d1fde5ce042a12e231ff032ae1aace89670ad15497672b28a6eb5699e9/gpiozero-1.6.2.tar.gz",
+                    "sha256": "0eb95a9db372146813276f92de7f43c883a3e9fe69597fc3d29c04ef3d5d5f9e"
                 }
             ]
         },
+        {
+            "name": "python3-site-packages",
+            "buildsystem": "simple",
+            "build-commands": [
+                "echo '/usr/lib/python3.9/site-packages/' > /app/lib/python3.9/site-packages/usr.pth"
+            ]
```

Comment:
this shouldn't be needed either.

Reply:
The custom Python is not able to use GTK in the runtime if this is missing